### PR TITLE
Update Docker image so it takes the ASTRO_API_TOKEN instead of client…

### PR DESCRIPTION
**Pull request by sgcg10**
… credentials

### CODE REVIEW

**Comment by sgcg10** [comment]
I did some testing directly using the TOKEN in Astronomer API and it works, so this should work

image

File: plugin/main.py [hunk 1]
```diff
@@ -171,8 +144,7 @@ def run(self, dry_run: bool = False):
         self._fetch_oauth_token()
```
Comment (Lowess) [inline plugin/main.py]: @sgcg10 you removed this function but not the call.

**Review by Lowess — CHANGES_REQUESTED** [review]
Please also update the README.md accordingly to keep it up to date with the recent changes. Also make sure you fully test the execution of the plugin locally following the instructions from https://github.com/Lowess/drone-astrocloud-deployer?tab=readme-ov-file#beginner-development

For testing the plugin you can redeploy the last astrocloud built image released here -> `https://<drone-url>/gumgum/mle-pipelines/617`

**Comment by Lowess** [comment]
> … credentials

Please make sure to rotate the credentials if it was leaked at any point, screenshots remain accessible even if you change them !

File: README.md [hunk 2]
```diff
@@ -37,16 +35,14 @@ pipeline:
       image: public.ecr.aws/gumgum-verity/oss/astro-ap-build:0.2.0
       <<: *astrocloud_env
       commands:
-        - docker login images.astronomer.cloud -u $${ASTRONOMER_KEY_ID} -p $${ASTRONOMER_KEY_SECRET}
+        - echo ${ASTRONOMER_API_TOKEN} | docker login images.astronomer.cloud -u _json_key --password-stdin
```
Comment (sgcg10) [inline README.md]: @Lowess just this part, I tried to execute the command docker login with the token secret in several ways but with no success, Do you have any idea on how to do it? We need to specify the username but we don't have one, I was trying to use the token id but you can never see it in the Astronomer GUI.

Comment (Lowess) [reply README.md]: This part does not need to change, please roll back. Astrocloud has changed the way the deployment is done not the way the docker image is pushed to their registry. Docker registry login should still continue using key id and key secret

**Review by Lowess — APPROVED** [review]

**Pull request merged**